### DISCUSSION TITLE
[Announcement] Homework 1 is out

### DISCUSSION THREAD

**Discussion by yeonhee-ryou** [prosyslab-classroom/cs348-information-security#468] [Announcements]
Hi all,
this is an announcement about homework 1.

Homework 1 is a programming assignment requiring you to implement encryption, decryption and crack algorithms of three classical cryptography.

Here is the GitHub classroom link for this homework:
https://classroom.github.com/a/sStKl2JF

Your implementation will be graded using GradeScope.
Here is the detailed instruction for GradeScope:
https://docs.google.com/document/d/1FWXTf4LI-DMvZIu2aRPnzGOzC7flR_G_8Bf9ZDGjX7s/edit?usp=sharing

Due: 03. 22 (Wed.) 23:59:59
Late Due: 03. 24 (Fri.) 23:59:59 ([late submission penalty](https://github.com/prosyslab-classroom/cs348-information-security#homework)).

Thanks.

**Comment by tncks0121** [comment]
Will the scores on Gradescope be the final score of the homework? There are "Total Points" (which is hidden now) and "Autograder Score" (which is shown now), and I'm curious if they are intended to be different (e.g. hidden tests)

**Comment by m-spitfire** [comment]
Can we submit on gradescope more than once or the submission on the site is the final one?

**Comment by tncks0121** [comment]
> Can we submit on gradescope more than once or the submission on the site is the final one?

You can submit on gradescope multiple times, I did it already

**Comment by m-spitfire** [comment]
Thanks for the quick reply @tncks0121!

**Comment by KihongHeo** [comment]
Yes. You can submit as many as you want. The final submission will be considered.

**Comment by yeonhee-ryou** [comment]
> Will the scores on Gradescope be the final score of the homework? There are "Total Points" (which is hidden now) and "Autograder Score" (which is shown now), and I'm curious if they are intended to be different (e.g. hidden tests)

The two scores are the same. The "Total Points" is a rounded value.

**Comment by ahnjonggun** [comment]
Will the name of the branch in my repository which will be submitted matter?

**Comment by yeonhee-ryou** [comment]
> Will the name of the branch in my repository which will be submitted matter?

It doesn't matter if your submission is graded well on GradeScope.
